pyproject.toml

```
@@ -41,6 +40,9 @@ full = [
     "python-lzo",
     "zstandard",
 ]
+crc32 = [
```

```suggestion
gcrc32 = [
```

Otherwise it may look like you don't get crc32 functionality at all.